### DISCUSSION TITLE
InterfaceVisualSFM

### DISCUSSION THREAD

**Discussion by mokrueger** [cdcseacave/openMVS#831] [Q&A]
Hi,

this might be a bit of a noob question. I have been using openMVS with COLMAP for a project before.
Since openMVS 2.0 the InterfaceVisualSFM has been removed from the apps/CMakeLists.txt
I've been wondering why that is.

**Comment by cdcseacave** [comment]
This has been discussed many times already, VisualSFM is inferior to most SFM pipelines, especially OpenMVG and COLMAP, plus the scene format it saves is not portable enough, even though many uses it for storing SfM scenes, and so when you get such a scene and try to use it in OpenMVS (or any other MVS pipeline) you quietly get inferior results due to imprecision in the format. SHOULD NOT BE USED! I will actually remove it completely since people are constantly trying to use it.
